docs/topics/jvm/jvm-spring-boot-add-db-support.md

```diff
@@ -36,7 +36,7 @@ class MessageService(val db: JdbcTemplate) {
 
 <deflist collapsible="true">
    <def title="Constructor argument and dependency injection – (val db: JdbcTemplate)">
-      <p>A class in Kotlin can have a primary constructor and one or more <a href="classes.md#secondary-constructors">secondary constructors</a>.
+      <p>A class in Kotlin has a primary constructor. It also can have one or more <a href="classes.md#secondary-constructors">secondary constructors</a>.
```

Comment:
```suggestion
      <p>A class in Kotlin has a primary constructor. It can also have one or more <a href="classes.md#secondary-constructors">secondary constructors</a>.
```